### PR TITLE
Add Trendy app

### PR DESCRIPTION
**Pull request by rhysd**
Trendy is an open source client for [GitHub trending repositories](https://github.com/trending).

http://rhysd.github.io/Trendy/

Repository page is below:

https://github.com/rhysd/Trendy

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Some good apps written with Electron.
 - [Kakapo](https://github.com/bluedaniel/kakapo-app) - An ambient sound mixer for relaxation or productivity.
 - [DevDogs](https://github.com/ragingwind/devdogs) - DevDocs.io API documentation.
 - [Blox Party](https://github.com/kvnneff/bloxparty) - Multiplayer Tetris game.
+- [Trendy](http://rhysd.github.io/Trendy/) - GitHub trending repositories client.
```
Comment (shinnn) [inline readme.md]: http://rhysd.github.io/Trendy/ -> https://rhysd.github.io/Trendy/

Comment (sindresorhus) [inline readme.md]: I would go with something like:

> Trending GitHub repos in your menubar.

Comment (rhysd) [inline readme.md]: @shinnn Thanks! I'll fix it.

@sindresorhus It sounds better!

**Comment by rhysd** [comment]
I fixed URL and description.

**Comment by sindresorhus** [comment]
Looks good! Bonus points for providing binaries for all platforms ;)

**Comment by rhysd** [comment]
Thanks :smile:

**Pull request merged**